client/components/banner/style.scss

```diff
@@ -8,37 +8,37 @@
 		padding-right: 48px;
 	}
 
-	@include banner-color( $blue-wordpress );
+	@include banner-color( var( --color-primary ) );
 
 	&.is-wpcom-plan {
 		&.is-upgrade-blogger {
-			@include banner-color(darken($alert-green, 20%));
+			@include banner-color( var( --color-success-dark ) );
 		}
 
 		&.is-upgrade-personal {
-			@include banner-color($blue-wordpress);
+			@include banner-color( var( --color-primary ) );
 		}
 
 		&.is-upgrade-premium {
-			@include banner-color($alert-yellow);
+			@include banner-color( var( --color-warning ) );
 		}
 
 		&.is-upgrade-business {
-			@include banner-color($orange-jazzy);
+			@include banner-color( $orange-jazzy );
```

Comment:
@drw158 in #28331 you used `--color-accent` for a replacement of `$orange-jazzy`; does that apply here as well?

Reply:
```suggestion
			@include banner-color( $muriel-orange-500 );
```